develop/reference_ddl.rst

```diff
@@ -294,6 +294,60 @@ Not-null constraints can be applied to any column (distribution or not) because
 
   ALTER TABLE ads ALTER COLUMN image_url SET NOT NULL;
 
+Using NOT VALID Constraints
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In some situations it can be useful to enforce constraints for new rows, while allowing existing non-conforming rows to remain unchanged. Citus supports this feature for CHECK constraints and foreign keys, using PostgreSQL's "NOT VALID" constraint designation.
+
+For example, consider an application which stores user profiles in a :ref:`reference table <reference_tables>`.
+
+.. code-block:: postgres
+
+   -- we'll store user emails case insensitively, so enable
+   -- the "citext" data type on the coordinator and workers
+   CREATE EXTENSION citext;
```

Comment:
Do we need to use citext for this example?

Reply:
No, we could also use text. I just thought it's a nice practice to encourage treating email addresses case-insensitively (citext indexes them that way and everything). CItext is included with postgres as one of those [additional supplied modules](https://www.postgresql.org/docs/11/contrib.html).

But if you think it's too distracting from the core lesson then I could remove it.